packages/kit/src/core/config/options.js

```diff
@@ -255,6 +255,11 @@ const options = object(
 
 			trailingSlash: list(['never', 'always', 'ignore']),
 
+			version: object({
+				name: string(null),
```

Comment:
```suggestion
				name: string(Date.now().toString()),
```

Reply:
But this will break the tests, because the timestamp will not be the same.

Reply:
I had it that way before.

Reply:
it might require some finessing (overriding/deleting the value during testing, a la `remove_keys`, or monkey-patching `Date.now()` or something) but i think it's probably preferable to be able to make guarantees about the shape of the config